File: opcua-webservice-spring/core/src/main/java/org/opcfoundation/webserver/service/message/digitaltwin/InvokeOperationResponse.java
package org.opcfoundation.webserver.service.message.digitaltwin;

import org.eclipse.milo.opcua.stack.core.types.builtin.Variant;

import java.util.List;

public class InvokeOperationResponse {
    private final List<Variant> outputArguments;

    public InvokeOperationResponse(List<Variant> outputArguments)
    {
        this.outputArguments = outputArguments;
    }

    public List<Variant> getOutputArguments() {
        return outputArguments;
    }
}

File: opcua-webservice-node/source/src/server/service/message/digitialtwin/InvokeOperationResponse.ts
import { UaVariant } from "opcua-webapi-ts";

export class InvokeOperationResponse {
    private readonly _outputArguments: Array<UaVariant>;

    constructor(outputArguments: Array<UaVariant>) {
        this._outputArguments = outputArguments;
    }

    get outputArguments(): Array<UaVariant> {
        return this._outputArguments;
    }
}